client/servers/manager.go

```diff
@@ -74,6 +74,16 @@ func (s *Server) String() string {
 	return s.addr
 }
 
+func (s *Server) Equal(o *Server) bool {
+	if s == nil && o == nil {
+		return true
+	} else if s == nil && o != nil || s != nil && o == nil {
+		return false
+	}
```

Comment:
Can be shortened to `if s == nil || o == nil { return s == o }`

h/t to @chelseakomlo